symengine/add.cpp

```diff
@@ -5,7 +5,9 @@
 #include <symengine/pow.h>
 #include <symengine/complex.h>
 #include <symengine/functions.h>
+#include <symengine/polynomial.h>
 
+using SymEngine::is_a;
```

Comment:
Is this needed? There are other instances(in the same file too) where `is_a<Mul>` etc., are used without `SymEngine::`